packages/js/src/executors/node/node.impl.ts

```diff
@@ -304,4 +306,23 @@ function runWaitUntilTargets(
   );
 }
 
+function fileToRunCorrectPath(fileToRun: string): string {
```

Comment:
@jaysoo I added this extra function here to make sure we get the correct path to the file. We cannot get the extension directly from the build options, because there's a number of things we do to get the extension for esbuild and rollup, so it's just easier to check the three possible extensions.